afqbrowser/site/client/js/tract-details.js

```diff
@@ -293,15 +320,23 @@ function ready(error, data) {
             }
         });
 
+
+
     // compute mean line
     afqb.plots.tractMean = d3.nest()
         .key(function (d) { return d.tractID; })
         .key(function (d) { return d.nodeID; })
         .rollup(function (v) {
-			return d3.mean(v, function (d) {
-				return +d[afqb.controls.plotsControlBox.plotKey];
-			});
-		})
+			 		return{
+						mean: d3.mean(v, function (d) {
+						 return +d[afqb.controls.plotsControlBox.plotKey];}),
+					  stderr: d3.deviation(v, function (d) {
+						 return +d[afqb.controls.plotsControlBox.plotKey];
+					   })/Math.sqrt(afqb.plots.tractData[0].values.length),

```

Comment:
@jyeatman The ready function uses n of the whole data set because we generate the plots prior to any split. Line 487 in tract-details.js (in updatePlots() function) uses the size of the first split group instead.